librocksdb-sys/build.rs

```diff
@@ -145,7 +145,8 @@ fn build_rocksdb() {
             .iter()
             .cloned()
             .filter(|file| match *file {
-                "port/port_posix.cc" | "env/env_posix.cc" | "env/io_posix.cc" => false,
+                "port/port_posix.cc" | "env/env_posix.cc" | "env/fs_posix.cc"
+                | "env/io_posix.cc" => false,
```

Comment:
can it just be `!file.ends_with("posix.cc")`?

Reply:
Actually can. But I prefer such approach. What if in the future will appear file with name which ends with `posix.cc` but relates to both OS ? We'll skip it.

Reply:
Right, but if a file name ends with posix.cc, it's almost certainly POSIX-specific. OTOH, it's easy to miss one such file when updating the sources as we can see here.

Reply:
It doesn't matter in the end.
The real problem is a lack of (required) windows CI check.

Reply:
@ordian exactly. By the way I'm trying to fix travis now 🙂